invisible_cities/detsim/simulate_S1_test.py

```diff
@@ -0,0 +1,95 @@
+import os
+import numpy as np
+
+from . simulate_S1 import generate_S1_time
+from . simulate_S1 import generate_S1_times_from_pes
+from . simulate_S1 import create_S1_waveforms
+from . simulate_S1 import compute_S1_pes_at_pmts
+from . simulate_S1 import create_lighttable_function
+from . simulate_S1 import binedges_from_bincenters
+
+from pytest import raises
+
+from hypothesis import given
+from hypothesis import settings
+from hypothesis.strategies import integers
+
+
+@settings(deadline=1000)
+@given(integers(min_value=0, max_value=10000))
```

Comment:
I think hypothesis is a total overkill for testing a size and a type of the output. If you do want to keep this test rename it to `test_generate_s1_time_output_type` and use some fixed size.